anyway/parsers/registered.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+import csv
+import glob
+import io
+import logging
+import os
+import re
+import sys
+from datetime import datetime
+
+import six
+from flask.ext.sqlalchemy import SQLAlchemy
+
+from ..models import RegisteredVehicle, City
+from ..utilities import init_flask, time_delta
+
+reload(sys)
```

Comment:
Why is reload needed?

Reply:
this is the only solution i found that actually work to deal with:
"UnicodeEncodeError: 'ascii' codec can't encode characters in position 65-67: ordinal not in range(128)" i will be very happy to replace it with non-hack solution, please advice :)